src/components/List/index.tsx

```diff
@@ -50,19 +70,30 @@ export const StyledListItem = styled(ListItem)<ListItemProps>`
 
 const StyledListSubItem = styled(ListItem)<ListItemProps>`
   &.MuiButtonBase-root.MuiListItem-root {
-    margin: 4px 0;
+    margin: 4px 0 4px 15px;
+    width: calc(100% - 15px);
+
+    &::before {
+      content: '';
+      width: 6px;
+      height: 1px;
+      background: #e1e1e1;
+      position: absolute;
+      left: -15px;
+    }
   }
 
   & .MuiListItemText-root span {
     line-height: 1;
   }
 
   &.MuiListItem-button:hover {
+    background-color: #f6f7f8;
```

Comment:
You're using this colour often as well. I'd export it from the variables file too.